### ISSUE TITLE
Missing Æ and Œ smallcaps

### ISSUE THREAD

**Issue by katef** [skosch/Crimson#34]
The `ae.sc` and `oe.sc` glyphs look like placeholders, and need to be drawn properly.

**Comment by katef** [comment]
In Roman, what's `ae.sc.alt01` supposed to be?

**Comment by skosch** [comment]
Not sure what that placeholder thing is, sorry. I don't remember making it, but then I don't remember much that happened five years ago.

I suppose an appropriate use would be a ligated small cap Æ, whereas the regular small cap is a broken-up AE. Thoughts?

**Comment by katef** [comment]
I don't think that would suit e.g. Danish, where I think Æ must be a ligature. My hunch is for both `ae` and `ae.sc` to be ligatures, else why provide them in the first place?

**Comment by skosch** [comment]
If I had to guess, I probably got the justification for breaking them up from some long gone ivory-tower Typophile thread. I don't recall what it was exactly.

You're absolutely correct about Scandinavian orthographies requiring them though, so it's Kate 1, Seb 0. (Well, probably more like 5-0, by now.)

**Comment by katef** [comment]
Us 6, World 0 :)

**Comment by katef** [comment]
Too heavy at the joints?

![oe](https://cloud.githubusercontent.com/assets/1371085/12175070/1aa636a0-b558-11e5-92c5-398b07d8b69e.png)

**Comment by skosch** [comment]
Maybe, by the slightest bit? I don't see anything wrong with it besides that though.
